Marketplace/voiceHUDApp/appResources/appData/voiceHUD_app.js

```diff
@@ -0,0 +1,836 @@
+                        "red": 0,
+                        "green": 255,
+                        "blue": 255
+                    },
+                    parentID: user.hudID[0]
+                }
+            ];
+            
+            hudProperties.forEach(function(element){   
+                user.hudID.push(Overlays.addOverlay("shape", element));   
+            });
+            audioBarProperties.forEach(function(element){   
+                user.audioBarIDs.push(Overlays.addOverlay("shape", element));   
+            });
+        }, 200);
+    }
+    
+
+    // This function deletes the HUD over a user
+    function deleteHUD(uuid) {
+        var user = userStore[uuid];
+        user.hudID.forEach(function(element){
+            if (element){
+                Overlays.deleteOverlay(element);
+            }
+        });
+        user.audioBarIDs.forEach(function(element){
+            Overlays.deleteOverlay(element);
+        });
+        user.hudID = [];
+        user.audioBarIDs = [];
+    }
+
+
+    // Update the local z-dimensions of the audio bars on the HUD 
+    // over a user to make it appear in synce with audio level
+    function updateHUD(uuid) {
+        var user = userStore[uuid];
+        Overlays.editOverlay(user.hudID[0], 
+            {rotation: Quat.multiply(Camera.orientation, Quat.fromPitchYawRollDegrees(90,0,0))});
+        user.userHeight = AvatarManager.getAvatar(uuid).getNeckPosition().y + 
+            AvatarManager.getAvatar(uuid).scale * AVATAR_HEIGHT_OFFSET;
+        Overlays.editOverlay(user.hudID[0], 
+            {position: {
+                x: AvatarManager.getAvatar(uuid).position.x, 
+                y: user.userHeight, 
+                z :AvatarManager.getAvatar(uuid).position.z
+            }});
+        if (user.audioLevel > 0.25){ 
+            user.hudID.forEach(function(id){
+                Overlays.editOverlay(id, { visible: true });
+            });
+            user.audioBarIDs.forEach(function(id){
+                Overlays.editOverlay(id, { visible: true });
+            });
+        } else {
+            setDefaultHUD(uuid);
+            Script.setTimeout(function(){
+                user.hudID.forEach(function(id){
+                    if (user.avgAudioLevel < 0.25){
+                        Overlays.editOverlay(id, { visible: false });
+                    }
+                });
+                user.audioBarIDs.forEach(function(id){
+                    if (user.avgAudioLevel < 0.25){
+                        Overlays.editOverlay(id, { visible: false });
+                    }
+                });
+            }, 5000);
+        }
+        if (user.audioLevel <= 0.5){ 
+            user.audioBarIDs.forEach(function(id){
+                Overlays.editOverlay(id, { color: {
+                    "red": 0,
+                    "green": 255,
+                    "blue": 0
+                } });
+            });
+        } else if (user.audioLevel > 0.5 && user.audioLevel <= 0.75){ 
```

Comment:
you don't need the `user.audioLevel > 0.5 &&` in here, because you will have already fallen into the above conditional if the value is <= 0.5